src/zenml/orchestrators/base_orchestrator.py

```diff
@@ -95,22 +128,7 @@ def supports_client_side_caching(self) -> bool:
 
 
 class BaseOrchestrator(StackComponent, ABC):
-    """Base class for all orchestrators.
-
-    In order to implement an orchestrator you will need to subclass from this
-    class.
-
-    How it works:
-    -------------
-    The `run(...)` method is the entrypoint that is executed when the
-    pipeline's run method is called within the user code
-    (`pipeline_instance.run(...)`).
-
-    This method will do some internal preparation and then call the
-    `prepare_or_run_pipeline(...)` method. BaseOrchestrator subclasses must
-    implement this method and either run the pipeline steps directly or deploy
-    the pipeline to some remote infrastructure.
-    """
```

Comment:
I would still keep an explanation here even though it changed a little bit.

Reply:
It's gonna be removed anyway, right? No one should ever look at this method from now on, and all the existing ones have already implemented it. IMO this explanation was useless anyway, reading it does not benefit anyway anyone doesn't already know what they're doing :D

Reply:
I think I only removed docstrings that we're either
- completely wrong, as they were copy pasted from a different orchestrator
- completely useless, as they were copy pasted from the base class (which as described above, is not only generic but also provides no actual guidelines on what to do)
- outdated, because it referenced some internal implementation details, which is not the purpose of a docstring.